src/main/java/FreedomBot/TickerFetcher.java

```diff
@@ -0,0 +1,87 @@
+package FreedomBot;
+
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.net.http.HttpResponse.BodyHandlers;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Base64;
+import javax.crypto.Mac;
+import javax.crypto.spec.SecretKeySpec;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+public class TickerFetcher {
+
+  public static void PrintTheTicker() {
+    try {
+      var key = "<your key>";
+      var secret = "<your secret>";
+      var passphrase = "<your passphrase>";
```

Comment:
As part of this PR, some way to let the user specify and safely/securely store these values needs to be devised